static/docs/user-guide/dvcignore.md

```diff
@@ -18,26 +19,126 @@ provide similar functionality as `.gitignore` files provide for `git`.
 - Each line should contain only one pattern;
 - During execution of commands that traverse directories, DVC will ignore
   matching paths;
-- Not every operation supports `.dvcignore`. To see current limitations, read
-  following paragraph.
+  
+## Remarks
+- Ignored files will not be saved in cache, they will be non-existent for DVC.
+  It's worth to remember that, especially when ignoring files inside 
+  DVC-handled directories. **It is crucial to understand, that DVC might remove
+  ignored files upon `dvc run` or `dvc repro`. If they are not produced by
+  [pipeline](/doc/get-started/pipeline) step, they can be deleted permanently.**
+- Keep in mind, that when you add to .dvcignore entries that affect one of the
+  existing <abbr>outputs</abbr>, its status will change and DVC will behave
+  as if that affected files were deleted;
+- If DVC stumbles upon `.dvcignore` file inside dependency or
```

Comment:
a dependency or an output

Reply:
Also not sure about phrase `stumbles upon`. Changed to `finds a` in c0aa902